pdp/static/js/pdp_raster_map.js

```diff
@@ -55,18 +97,23 @@ function dasToUnitsSince(data) {
         dateString = m[3],
         sDate;
 
+    var calendar;
+    reg = /calendar \"(standard|gregorian|365_day|noleap|360_day)\"/,
+    m = reg.exec(s),
+    calendar = m[1];
```

Comment:
Last change request, I swear! Line 103 will fail (and stop the script) in the event that the DAS responds with a calendar that is not one of those listed. E.g.:

```javascript
> var s = "calendar \"foo\"";
undefined
> reg = /calendar \"(standard|gregorian|365_day|noleap|360_day)\"/,
... m = reg.exec(s),
... calendar = m[1];
TypeError: Cannot read property '1' of null
    at repl:3:13
    at realRunInThisContextScript (vm.js:22:35)
    at sigintHandlersWrap (vm.js:98:12)
    at ContextifyScript.Script.runInThisContext (vm.js:24:12)
    at REPLServer.defaultEval (repl.js:346:29)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.onLine (repl.js:545:10)
    at emitOne (events.js:101:20)
    at REPLServer.emit (events.js:188:7)
```
Please ensure that a default value (standard) is assumed here as well and be robust against possible unknown input.